### PR TITLE
Daily metrics

### PR DESCRIPTION
**Pull request by perrywilson**
Added throttling lambda to age off daily metrics in Redis. Added the Onboarding.md documentation for new developers. 

Related to https://github.com/jhuapl-boss/boss/pull/59

### CODE REVIEW

File: cloud_formation/lambda/cache_throttle/index.py [hunk 1]
```diff
@@ -13,13 +13,23 @@
 # limitations under the License.
 
 import redis
+from datetime import datetime, timedelta
```
Comment (movestill) [inline cloud_formation/lambda/cache_throttle/index.py:16]: Can you document the expected keys in the `event` dict?

File: docs/Onboarding.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+1. An AWS IAM account on [thebossdev](https://thebossdev.signin.aws.amazon.com/console) with credentials that enable aws cli use
+2. Access to the [APL Microns](https://github.com/aplmicrons) repository. The new user will need a bosslet config created and the ability to download the bosslet-configs project.
+3. Access to the [JHUAPL Boss](https://github.com/jhuapl-boss) repository to pull and make code commits.
+
+# Development Envrionment
+The [Installation Guide](./InstallGuide.md) is geared to installing BossDB in new AWS subscription. It covers many of the topics you won't need to get into right now. This **onboarding guide** will highlight what you need to do to get up and running quickly.
+
+## Choose your Python Distribution
+This guide assumes you have installed anaconda for python 3. You can download the installer for your platform from [Anaconda](https://www.anaconda.com/products/individual).
```
Comment (movestill) [inline docs/Onboarding.md:14]: How about add verbiage saying that any Python virtual environment may be used?

**Comment by perrywilson** [comment]
Please review updates. I am done making changes

File: docs/Onboarding.md (hunk 2 above)
Comment (sandyhider) [reply docs/Onboarding.md:14]: Added Verbiage

**Review by sandyhider — APPROVED** [review]

**Pull request merged**